projects/1.18/assets/caged-mobs/cagedmobs/lang/zh_cn.json

```
@@ -1,51 +1,78 @@
+	"item.cagedmobs.cooking_upgrade.info": "自动烹饪可烹饪的生物掉落物",
+	"item.cagedmobs.lightning_upgrade.info": "用闪电杀死笼中的生物",
+	"item.cagedmobs.arrow_upgrade.info": "用箭矢杀死笼中的生物",
+	"item.cagedmobs.experience_upgrade.info": "笼中的生物被杀死时，会掉落经验宝珠",
+	"item.cagedmobs.looting_upgrade.info": "在笼子掉落物上应用抢夺 III",
+	"item.cagedmobs.creative_upgrade.info": "使生物立刻长大",
+	"item.cagedmobs.creative_upgrade.info2": "创造模式限定",
 
 	"item.cagedmobs.upgrades.attach": "可以安装到生物笼上",
+	"item.cagedmobs.upgrades.stack": "升级可以堆叠",
 
 	"item.cagedmobs.dragon_scale": "龙鳞",
-	"item.cagedmobs.sponge_fragment": "海绵碎片",
 	"item.cagedmobs.nether_star_fragment": "下界之星碎片",
+	"item.cagedmobs.warden_receptor": "监守者的感受器",
+	"item.cagedmobs.sponge_fragment": "海绵碎片",
 	"item.cagedmobs.honey_drop": "蜂蜜滴",
+	"item.cagedmobs.milk_drop": "牛奶滴",
+	"item.cagedmobs.experience_orb": "经验宝珠",
+	"item.cagedmobs.empty_spawn_egg": "空的刷怪蛋",
 	"item.cagedmobs.star_infused_netherite_ingot": "注星下界合金锭",
 	"item.cagedmobs.star_infused_netherite_nugget": "注星下界合金粒",
+
+	"item.cagedmobs.experience_orb.info": "右击消耗",
+	"item.cagedmobs.experience_orb.info2": "Shift+右击消耗全部",
 	
-	"block.cagedmobs.mobcage": "生物笼",
-	"block.cagedmobs.hoppingmobcage": "漏式生物笼",
+	"block.cagedmobs.mob_cage": "生物笼",
+	"block.cagedmobs.hopping_mob_cage": "漏式生物笼",
 	"block.cagedmobs.star_infused_netherite_block": "注星下界合金块",
-	"block.cagedmobs.mobcage.envNotSuitable": "笼中的环境不适合该生物。",
-	"block.cagedmobs.mobcage.mainInfo": "可以根据存储的DNA样本去复制生物的掉落物",
-	"block.cagedmobs.mobcage.envInfo": "需要内设一处适合生物的环境",
-	"block.cagedmobs.mobcage.hoppingInfo": "产生的掉落物将放入下方容器中",
-	"block.cagedmobs.mobcage.upgrading": "可以安装升级",
-	"block.cagedmobs.mobcage.hoppingCagesDisabled1": "已禁用自动收获。",
-	"block.cagedmobs.mobcage.hoppingCagesDisabled2": "漏式生物笼会像普通生物笼那样运作。",
-	"block.cagedmobs.mobcage.requiresWater": "该生物需要一个含水的生物笼。",
-	"block.cagedmobs.mobcage.envRequired": "你需要先打造一处环境。",
-	"block.cagedmobs.mobcage.samplerAlreadyUsed": "该采样器已含有另一个生物的DNA。",
-	"block.cagedmobs.mobcage.samplerNotSufficient": "该采样器的级别不足以取回笼中生物的DNA样本。",
-	"block.cagedmobs.starinfusednetheriteblock.beacon": "可以用作信标基座方块。",
+	"block.cagedmobs.mob_cage.envNotSuitable": "笼中的环境不适合该生物。",
+	"block.cagedmobs.mob_cage.mainInfo": "可以根据储存的DNA样本去复制生物的掉落物",
+	"block.cagedmobs.mob_cage.envInfo": "需要内设一处适合生物的环境",
+	"block.cagedmobs.mob_cage.hoppingInfo": "产生的掉落物将放入下方容器中",
+	"block.cagedmobs.mob_cage.upgrading": "可以安装升级",
+	"block.cagedmobs.mob_cage.rightClickHarvest": "持剑右击以收获",
+	"block.cagedmobs.mob_cage.hoppingCagesDisabled1": "已禁用自动收获。",
+	"block.cagedmobs.mob_cage.hoppingCagesDisabled2": "漏式生物笼会像普通生物笼那样运作。",
+	"block.cagedmobs.mob_cage.requiresWater": "该生物需要一个含水的生物笼。",
+	"block.cagedmobs.mob_cage.envRequired": "你需要先打造一处环境。",
+	"block.cagedmobs.mob_cage.cageAlreadyUsed": "该生物笼已含有另一个生物的DNA。",
+	"block.cagedmobs.mob_cage.samplerNotSufficient": "该采样器的级别不足以取回笼中生物的DNA样本。",
+	"block.cagedmobs.mob_cage.upgradesFull": "该生物笼已经装满了升级！",
+	"block.cagedmobs.mob_cage.spawnEggsDisabled": "刷怪蛋用途已被禁用。",
```

```suggestion
	"block.cagedmobs.mob_cage.spawnEggsDisabled": "已禁用刷怪蛋。",
```
```suggestion
	"block.cagedmobs.mob_cage.spawnEggsDisabled": "已禁用刷怪蛋功能。",
```
普通的句子永远比把字句/被字句好
不过没啥大问题，如果觉得有歧义可以不改

---

这里是刻意区分的，我当然看过翻译准则，尽量不使用被动句
这句话是在 `配置中禁用刷怪蛋用途的情况` 下手持刷怪蛋右击生物笼弹出的红色提示信息，我考虑到如果改成 `已禁用刷怪蛋（功能）。` 的语序，就像是在暗示因为 `手持刷怪蛋右击生物笼` 而导致 `刷怪蛋（功能）` 的禁用，而事实是它是通过配置文件禁用的

---

ok